include/ibm/management_console_rest.hpp

```diff
@@ -1075,6 +1160,14 @@ inline void requestRoutes(App& app)
                 handleFileUrl(req, asyncResp, fileName);
             });
 
+    BMCWEB_ROUTE(app, "/ibm/v1/Host/ConfigFiles")
```

Comment:
Why before were we posting to /ibm/v1/Host/ConfigFiles/<str> ?

Reply:
So Code update is doing multipart at 

    "MultipartHttpPushUri": "/redfish/v1/UpdateService/upload",
    
    Does it make sense for us to follow?

Reply:
Should we advertise this in the GET somehow like Code update?

Reply:
There is already a GET and PATCH  command implemented on this URL for the save area file upload and download purpose. This is a new POST command to upload multiple files at one shot. 
Code update is the redfish path. This /ibm/v1 is a REST way of implementing the routes.

Reply:
I would prefer we match code update here...
"MultipartHttpPushUri": "/redfish/v1/UpdateService/upload",